cos/models/Game.py

```diff
@@ -25,15 +25,15 @@ class Game(object):
     # Maximum Players allowed in the game
     PLAYER_MAX = 4
     # Options for player colors
-    PLAYER_COLORS = ["white", "blue", "orange", "red"]
+    PLAYER_COLORS = ["blue", "white", "orange", "red"]
 
     def __init__(self, name):
         """ Constructor for Game class:
                 - no required params
                 - sets private attributes of game_id (random string of alpaha-numeric chars and
                   creates first player object named "Player1" 
         """
-        id_string = ''.join(random.choice('0123456789ABCDEFGHIJKLMNZQRSTUVWXYZ') for i in range(6))
+        id_string = "GAME1" #''.join(random.choice('0123456789ABCDEFGHIJKLMNZQRSTUVWXYZ') for i in range(6))
```

Comment:
This also should not be hard coded as it breaks the rest of the code for multiple games.